stl/inc/yvals_core.h

```diff
@@ -1285,7 +1293,7 @@
 #define __cpp_lib_constexpr_dynamic_alloc 201907L
 #define __cpp_lib_constexpr_functional    201907L
 #define __cpp_lib_constexpr_iterator      201811L
-#define __cpp_lib_constexpr_memory        201811L
+#define __cpp_lib_constexpr_memory        202202L

```

Comment:
Currently this PR bumps `__cpp_lib_constexpr_memory` in C++20 mode, which may be incorrect.